scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -312,8 +315,8 @@ task LoadData {
       sample_name="${SAMPLE_NAMES_ARRAY[$i]}"
 
       # we always do our own localization
-      gsutil cp $gs_input_vcf input_vcf_$i.vcf.gz
-      gsutil cp $gs_input_vcf_index input_vcf_$i.vcf.gz.tbi
+      gsutil ~{"-u " + billing_project_id} cp $gs_input_vcf input_vcf_$i.vcf.gz
+      gsutil ~{"-u " + billing_project_id} cp $gs_input_vcf_index input_vcf_$i.vcf.gz.tbi
```

Comment:
This is a little tricky, may need to discuss with faces. Also I am leaning on my Cromwell experience which may or may not be the same as the behavior we want here:

If `-u billing_project_id` is specified, the specified billing project will be charged for egress regardless of whether the bucket is requester pays or not. This may lead to users being charged more than they expect to be if they supply a billing project id and the source bucket is actually not requester pays.

At least as of several years ago there wasn't a "nice" way to tell if a bucket was requester pays or not, hence localization code [like this](https://github.com/broadinstitute/cromwell/blob/a680a3560cbff93973437e849632b1481527f35e/supportedBackends/google/pipelines/v2beta/src/main/resources/gcs_transfer.sh#L124-L138) in Cromwell. 

- Try to localize the file initially without requester pays, using `$NO_REQUESTER_PAYS_COMMAND`. If that works, great, we're done.
- If that doesn't work, look in the "gsutil_log" log file to see if there's an error message that looks like a requester pays error. If there is such a message, set up to retry with the `$REQUESTER_PAYS_COMMAND`.

Loads of retry logic wrapping all of this which wouldn't be the worst idea for us to have here but is certainly not within the scope of this ticket.

Reply:
I updated the documentation to explain that passing this input will charge it for egress, and that the bucket being requester pays would be one reason why to do so.

Reply:
That seems like a reasonable solution to this, thanks for calling that out Miguel.